src/archetypes/referencebrowserwidget/skins/referencebrowser/referencebrowser.js

```diff
@@ -447,6 +447,17 @@ jQuery(function ($) {
         });
     }
 
+    $(document).on('change', '#indexSelector', function (event) {
+        if ($.inArray(this.value, eval($('#searchWildcardHelp').data()['wildcardableindexes'])) === -1) {
```

Comment:
Hi @vincentfretin 

OK, I updated my PR to pass valid JSON to jQuery.data so it is extracted automatically.

The problem was that by default, passed data was someting like :
"['Title', 'SearchableText']" and valid JSON is '["Title", "SearchableText"]'...

This time it works without eval() ;-)

What do you think now?

Gauthier